cookbooks/aws-parallelcluster-scheduler-plugin/resources/set_user_privileges.rb

```diff
@@ -28,7 +28,7 @@
     Chef::Log.info("Set sudo privileges for #{node['cluster']['scheduler_plugin']['user']}")
     # Ensure scheduler user has sudoers capability
     template '/etc/sudoers.d/99-parallelcluster-scheduler-plugin' do
-      source 'byos_user/99-parallelcluster-scheduler-plugin.erb'
+      source 'scheduler_plugin_user/99-parallelcluster-scheduler-plugin.erb'
```

Comment:
this is addressed in https://github.com/aws/aws-parallelcluster-cookbook/pull/1227